ConsoleApp1/adminScherm.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.IO;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using CinemaUI;
+using CinemaUI.Builder;
+namespace CinemaApplication
+{
+    partial class Program
+    {
+        static Window adminScherm = new Window(true);
+        static void selectieSchermAdmin()
```

Comment:
Probeer dit een makkelijkere naam te geven. Misschien AdminScherm. Zo weet je dat elke functie een window heeft in pascal case